crates/router/src/connector/checkout/transformers.rs

```diff
@@ -140,20 +140,14 @@ pub enum CheckoutPaymentStatus {
     Captured,
 }
 
-impl From<transformers::Foreign<(CheckoutPaymentStatus, Option<enums::CaptureMethod>)>>
+impl From<transformers::Foreign<(CheckoutPaymentStatus, i32)>>
     for transformers::Foreign<enums::AttemptStatus>
 {
-    fn from(
-        item: transformers::Foreign<(CheckoutPaymentStatus, Option<enums::CaptureMethod>)>,
-    ) -> Self {
-        let item = item.0;
-        let status = item.0;
-        let capture_method = item.1;
+    fn from(item: transformers::Foreign<(CheckoutPaymentStatus, i32)>) -> Self {
```

Comment:
Could you update this implementation signature to accept the `Balances` struct instead of an `i32`? Passing in an `i32` in a tuple doesn't really provide much information about what that int is and we are required to search for usages of this `From` implementation to figure out what value has to be passed.